src/main/java/TaskStorage.java

```diff
@@ -0,0 +1,59 @@
+
+import java.util.ArrayList;
+
+/**
+ * A storage of Tasks.
+ */
+class TaskStorage {
+    /**
+     * An ArrayList of Tasks.
+     */
+    ArrayList<Task> storage = new ArrayList<>(100);
+
+    /**
+     * Lists all the tasks stored.
+     */
+    public void listTasks() {
+        String topDivider = "~~~~~~~~~~~~~~~~o~~~~~~~~~~~~~~~~\n" + "Duke's Response: \n"
+                    + "Here are the tasks in your list: \n";
+        String botDivider = "~~~~~~~~~~~~~~~~o~~~~~~~~~~~~~~~~";
+        System.out.println(topDivider);
+
+        for (int i = 0; i < this.storage.size(); i++) {
+            String output = this.storage.get(i).provideDetails();
+            System.out.println((i + 1) + "." + output);
+        }
+
+        System.out.println(botDivider);
+    }
+
+
+    /**
+     * Stores a new task in storage.
+     * @param task The task to be stored.
+     */
+    public void storeTasks(Task task) {
+        this.storage.add(task);
+    }
+
+    /**
+     * Updates the status of a Task.
+     * @param number The number representing the task to be updated.
+     */
+    public void updateTask(int number) {
```

Comment:
Can we change the parameter name? The parameter name can be `index` to make it more self explanatory, similar for the rest below.

Reply:
> Good effort 😊 Here is a some general overview:
> 
> * Good effort to separate the different tasks into separate components
> * Can try to choose method/variable names that are self-explanatory
> * Avoid unnecessary line breaks/indentations
> 
> Moving forward, you can also try to extract the different components into their own class, i.e. `Parser`, `Command`, `Ui`
> 
> All the best! 🔥

Thank you for your suggestions on using better names for variables according to Java conventions, the appropriate use of blank lines, and also suggesting the use of switch-case instead of if-else statements. My code at this instance was not well-refined yet. Thanks again for the comments!